tests/sentinel/tests/06-ckquorum.tcl

```diff
@@ -24,9 +24,14 @@ test "CKQUORUM detects failover authorization cannot be reached" {
         kill_instance sentinel [expr {$i + 1}]
     }
 
-    after 5000
-    catch {[S 0 SENTINEL CKQUORUM mymaster]} err
-    assert_match "*NOQUORUM*" $err
+    # We need to make sure that other sentinels are in `DOWN` state
+    # from the point of view of S 0 before we executing `CKQUORUM`.
+    wait_for_condition 300 50 {
+        [catch {S 0 SENTINEL CKQUORUM mymaster} e] == 1
```

Comment:
If e is not used, it can be omitted.

```suggestion
        [catch {S 0 SENTINEL CKQUORUM mymaster}] == 1
```

... But why not check e after finished waiting? I.e. keep the `assert_match "*NOQUORUM*" $err`

Reply:
right, we can remove the `e`
> ... But why not check e after finished waiting? I.e. keep the assert_match "*NOQUORUM*" $err

because `[catch {S 0 SENTINEL CKQUORUM mymaster}] == 1` it also make sure the result is `assert_match "*NOQUORUM*" $err`

but i agree we keep the `assert_match "*NOQUORUM*" $err` is more better. we can make sure the error message is exactly `NOQUORUM` (for now, if an error happened, it is always `NOQUORUM`)

Reply:
> (for now, if an error happened, it is always `NOQUORUM`)

Ok, i think it's not important to check the error message then (for now)